docs/conf.py

```diff
@@ -250,4 +250,8 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {
+    "python": ('http://docs.python.org/', None),
+    "persistent": ('https://persistent.readthedocs.io/en/latest/', None),
+    "ZODB": ("http://www.zodb.org/en/latest/", None),
```

Comment:
Can we use https:// everywhere?

Reply:
Done for python.org, but there is a problem with https://www.zodb.org/

Reply:
[Bug reported](https://github.com/zopefoundation/ZODB/issues/284)

Reply:
Maybe we want to use https://zodb.readthedocs.io/en/latest/ instead?

Reply:
Well, my understanding is that www.zodb.org is the official documentation, and zodb.readthedocs.io is a community book based on [this repository](https://github.com/cguardia/ZODB-Documentation), the latter received few love lately. The last commit is 4 years old, and most activity stop since 2011.

Reply:
Oops, how confusing!  I wish it was called zodb-book.rtd.org or something then.